### ISSUE TITLE
SlevomatCodingStandard.Namespaces.UnusedUses does not recognize namespaces used as a "base"

### ISSUE THREAD

**Issue by oojacoboo** [slevomat/coding-standard#1178]
The following fails, as it doesn't consider `Foo\Bar` as having been "used" for `use Foo`.

`SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse`

```php
<?php

use Foo;

class Hello
{
    public function speak(): string
    {
        $bar = new Foo\Bar();
        return $bar->blah();
    }
}
```

**Comment by kukulich** [comment]
@ocean90 What version do you use? Have you tried `dev-master` ?

**Comment by oojacoboo** [comment]
Sorry, that's `6.3.10`.  We haven't run `dev-master` as we have other sniffs and deps that have always been a bit of a juggle.

**Comment by kukulich** [comment]
`6.3.10` is not even latest release...

**Comment by oojacoboo** [comment]
It does seem to be fixed in 6.4.1.  Sorry about that, closing this out.  Thanks.
